Add Consolidated IC job expectation (#15)
holaluz/handbook: organization/roles/consolidated_individual_contributor_role.md

    * Contribute to Holaluz technology by consolidating basic and standards for own discipline best practices.
    * Participate in spikes discussions with the aim of understanding but also contributing to new solutions 
    * Support Holaluz operability  in order to guaranty business key processes by:
      * helping to increase efficiency and enabling the automation of processes.
      * Helping to maintenance of reliable information systems.
      * Bugfixing non right functionalities according to the complexity by  identifying errors and proposing solutions, or helping and supporting more experienced people in finding the best solutions for more complex issues.
      * Contribute in outages remediation, resolution and prevention via postmortems, improving monitoring, proactive testing, etc.
    * Review work done in domains you’re learning or you already master.
    * According to projects complexity, document processes by drafting manuals or/and guides with the aim of describe projects, steps required and tasks need to be followed.

  * **Be eager to learn**

>     * Actively pursue improving and consolidating higher engineering skills  (including but not limited writing readable/testable/efficient code, mastering technology in our stack), through guidance and mentorship, or though day-to-day work as; pairing with more senior people, bots reviews, pulls request and code reviews from more experienced developers.

Reviewer: ```suggestion
    * Actively pursue improving and consolidating higher engineering skills  (including but not limited writing readable/testable/efficient code, mastering technology in our stack), through guidance and mentorship, or though your daily work as; pairing with the other developers,  pulls request and code reviews from your developer peers.
```

I do believe that there is no need to do some pair-programming with more experienced/senior developers in order to improve the programming skills or master the stack. It's the given feedback and the different point of views that actually leads towards being a better developer.

Reviewer 2: Agree! But with "pairing with the other developers" or "pairing with other developers"?

Reviewer: I would go for `pairing with the other developers`
